### DISCUSSION TITLE
XMC database

### DISCUSSION THREAD

**Discussion by arieslaptop** [nofeletru/UsbAsp-flash#75] [Q&A]
Dear sir, kindly add xmc support in import.ini for the import database. 
![image](https://user-images.githubusercontent.com/88767405/215763371-429f5b59-3123-435e-b869-eab0e9da3210.png)
![image](https://user-images.githubusercontent.com/88767405/215763519-3356eadd-53c7-41c2-bcc6-0595f9e707fe.png)

**Comment by nofeletru** [comment]
Done.
